### PR TITLE
fix: display link as inline-flex

### PR DESCRIPTION
**Pull request by Averethel**
## Before

Screenshot 2022-10-06 at 12 39 23

## After

Screenshot 2022-10-06 at 12 39 32

### CODE REVIEW

**Review by lkappeler — APPROVED** [review]
Thanks for taking care. (Did not check the icons but I assume they still work with inline flex)
@GutierrezGo I think this solves the issue I pinged you about in the other PR

File: src/themes/components/link.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { ComponentMultiStyleConfig, SystemStyleObject } from '@chakra-ui/react';
 const linkBaseStyle: SystemStyleObject = {
   textStyle: 'body',
   color: 'blue.700',
-  display: 'flex',
+  display: 'inline-flex',
   flexDirection: 'row',
```
Comment (GutierrezGo) [inline src/themes/components/link.ts:7]: I'm wondering if we still need the row direction here.

Comment (Averethel) [reply src/themes/components/link.ts:7]: dang, I've only seen that comment after merging. `row` is the default as far as I remember. But defaults can change, doesn't hurt to have it.

Comment (GutierrezGo) [reply src/themes/components/link.ts:7]: oki, doki.
Thanks for jumping on the fix ❤️

**Pull request merged**